catalogue/webapp/components/ConfirmItemRequest/ConfirmItemRequest.tsx

```diff
@@ -138,8 +138,8 @@ const ConfirmedDialog: FC<ConfirmedDialogProps> = ({ currentHoldNumber }) => (
       />
     </Header>
     <p>
-      Your request will be available to pick up from the library (Rare Materials
-      Room, 3rd Floor) for two weeks.
+      It will be available to pick up from the library (Rare Materials Room,
+      level 3) for two weeks.
```

Comment:
I was just wondering if we have a house style of putting `Level` in uppercase? @DominiqueMarshall cc.  @dannybirchall maybe?

Reply:
@davidpmccormick lower case l, as here:

https://wellcomecollection.org/access